src/find-options/FindOneOptions.ts

```diff
@@ -24,7 +18,7 @@ export interface FindOneOptions<Entity = any> {
     /**
      * Specifies what columns should be retrieved.
      */
-    select?: FindOptionsSelect<Entity> | FindOptionsSelectByString<Entity>
+    select?: FindOptionsSelect<Entity>
```

Comment:
this is a breaking change. You should not remove this type signature, you'll break other people code (including people who use RDBMSes)

Reply:
Done 👍